tests/test_classifier/test_confusion_matrix.py

```diff
@@ -61,3 +60,28 @@ def test_one_class(self):
         cm = ConfusionMatrix(model, classes=[0])
         cm.fit(self.X_train, self.y_train)
         cm.score(self.X_test, self.y_test)
+
+    def test_defined_mapping(self):
+        model = LogisticRegression()
+        classes = ['one', 'two', 'three', 'four', 'five', 'six', 'seven', 'eight', 'nine']
+        mapping = {'zero': '0', 'one': '1', 'two': '2', 'three': '3', 'four': '4', 'five': '5',
+                   'six': '6', 'seven': '7', 'eight': '8', 'nine': '9'}
+        cm = ConfusionMatrix(model, classes=classes, label_mapping = mapping)
+        cm.fit(self.X_train, self.y_train)
+        cm.score(self.X_test, self.y_test)
+
+    def test_inverse_mapping(self):
+        model = LogisticRegression()
+        classes = ['zero', 'one', 'two', 'three', 'four', 'five', 'six', 'seven', 'eight', 'nine']
+        cm = ConfusionMatrix(model, classes=classes)
+        cm.fit(self.X_train, self.y_train)
+        cm.score(self.X_test, self.y_test)
+
+    def test_isclassifier(self):
+        model = PassiveAggressiveRegressor()
```

Comment:
Hehe, I had no idea a `PassiveAggressiveRegressor` existed ... that's quite the name!